streaming.md

```diff
@@ -143,7 +144,7 @@ message OutputFrame {
 ### OutputFrame
 `OutputFrames` represent data coming out of the function. Upon reception of a result data structure on the j-th output stream of a function, an invoker
 1. MUST serialize the data structure to a series of bytes (hereafter the `OutputFrame payload`) by selecting a MIME type that is compatible with the preference ordered list of MIME types specified in the j-th string of `expectedContentTypes` seen in the [`StartFrame`](#StartFrame). As a consequence, the invoker MUST not send any `OutputFrames` until it has received the `StartFrame`. 
-2. If it cannot successfully serialize data using such a MIME type, then it MUST fail this rpc invocation
+2. If it cannot successfully serialize data using such a MIME type, then it MUST fail this rpc invocation with a crafted status error of code [InvalidArgument](https://github.com/grpc/grpc/blob/2a7191abe717d3ce33b5fcbfd6bd118367499cde/doc/statuscodes.md#status-codes-and-their-use-in-grpc) and the error message MUST start with `"Invoker: Not Acceptable"`. An invoker MAY append error details to that message prefix.
```

Comment:
How should error details be appended? For instance, should there be punctuation and/or whitespace, e.g. `"Invoker: Not Acceptable: error details"` or is it ok to append the details without any punctuation or whitespace, e.g. `"Invoker: Not Acceptableerror details"`?

Reply:
What should the invoker do if serialisation fails?

Reply:
1) should add punctuation, but code shouldn't try to chop the prefix IMO
2) the sentence indeed needs to be changed slightly to distinguish between unsupported MIME type(s) and error happening during serialization (two distinct cases). The latter can let the generic gRPC failure bubble up